src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -183,7 +187,9 @@ public int createParquetFiles(
       String destinationDataSourceName,
       String controlFileDataSourceName,
       String ingestTableName,
-      Optional<Integer> csvSkipLeadingRows)
+      Integer csvSkipLeadingRows,
```

Comment:
now that you know it can't be null, you can use `int`
```suggestion
      int csvSkipLeadingRows,
```

Reply:
Unfortunately, we still pass this parameter in for JSON ingests, and it can be null for that.

Reply:
ah, of course :)